src/datachain/data_storage/warehouse.py

```diff
@@ -404,13 +404,12 @@ def dataset_stats(
         expressions: tuple[_ColumnsClauseArgument[Any], ...] = (
             sa.func.count(table.c.sys__id),
         )
-        if "file__size" in table.columns:
-            expressions = (*expressions, sa.func.sum(table.c.file__size))
-        elif "size" in table.columns:
-            expressions = (*expressions, sa.func.sum(table.c.size))
+        for c in table.columns:
+            if c.name.endswith("file__size"):
+                expressions = (*expressions, sa.func.sum(c))
         query = select(*expressions)
         ((nrows, *rest),) = self.db.execute(query)
-        return nrows, rest[0] if rest else None
+        return nrows, sum(rest) if rest else 0
```

Comment:
We are waiting for `int` in many places, errors happens if numbers of rows is set, but size is `None` (for example, [here](https://github.com/iterative/datachain/pull/418/files#diff-3b39a77247b3274508f28e3a5263a04189e57183b5bcfade572ccef288d1f293R1437-R1440)). It is better to return `0` in such cases.

Reply:
Error looks like this:

```
$ datachain dataset-stats laion_pq
Number of objects: 2
Error: bad operand type for abs(): 'NoneType'
$
```